### PR TITLE
[Rest Server] fix fetch job status failed

### LINKED ISSUE #4330: rest-server reports: fetch job status failed: podCompletionStatus.containers is not iterable
How to reproduce:

1. Setup a hived vc which has more cpu capacity than ground truth. For example, you have a node with 10 CPUs, but configure the hived as it has 20 CPUs.
2. Submit a job to that vc, which takes all CPUs of the vc capacity.

Then try to access the submitted job:

image

It cannot be stopped:

image

### PR DESCRIPTION
**Pull request by Binyang2014**
fix #4330

### CODE REVIEW

File: src/rest-server/src/models/v2/job/k8s.js [hunk 1]
```diff
@@ -1093,7 +1093,10 @@ const generateExitDiagnostics = (diag) => {
 };
 
 const extractRuntimeOutput = (podCompletionStatus) => {
-  if (_.isEmpty(podCompletionStatus)) {
+  if (
+    _.isEmpty(podCompletionStatus) ||
```
Comment (abuccts) [inline src/rest-server/src/models/v2/job/k8s.js]: what do you want to check for `podCompletionStatus`, isn't `if (podCompletionStatus)` enough

File: src/rest-server/src/models/v2/job/k8s.js [hunk 2]
```diff
@@ -1093,7 +1093,10 @@ const generateExitDiagnostics = (diag) => {
...
+    !_.isArray(podCompletionStatus.containers)
```
Comment (abuccts) [inline src/rest-server/src/models/v2/job/k8s.js]: why not use built-in `Array.isArray` in es6

File: src/rest-server/src/models/v2/job/k8s.js (hunk 1 above)
Comment (Binyang2014) [reply src/rest-server/src/models/v2/job/k8s.js]: Just want to check for `null` or `undefined`. Will change to use simple `if` sentence

File: src/rest-server/src/models/v2/job/k8s.js (hunk 2 above)
Comment (Binyang2014) [reply src/rest-server/src/models/v2/job/k8s.js]: Good to known that

Comment (Binyang2014) [reply src/rest-server/src/models/v2/job/k8s.js]: Fixed

File: src/rest-server/src/models/v2/job/k8s.js (hunk 1 above)
Comment (Binyang2014) [reply src/rest-server/src/models/v2/job/k8s.js]: Fixed

**Pull request merged**